core/src/main/scala/org/broadinstitute/dsde/rawls/jobexec/WorkflowSubmissionActor.scala

```diff
@@ -186,6 +186,25 @@ trait WorkflowSubmission extends FutureSupport with LazyLogging with MethodWiths
     }
   }
 
+  def getRuntimeOptions(googleProjectId: String, bucketName: String)(implicit executionContext: ExecutionContext): Future[Option[JsValue]] = {
+    for {
+      regionOption <- googleServicesDAO.getRegionForRegionalBucket(bucketName)
+      runtimeOptions <- {
+        regionOption match {
+          case Some(region) =>
+            for {
+              zones <- googleServicesDAO.getComputeZonesForRegion(GoogleProjectId(googleProjectId), region)
+            } yield {
+              Option(JsObject("zones" -> JsString(zones.mkString(" "))))
+            }
+          case None =>
+            // if the location is `multi-region` we want the default zones from config
+            Future.successful(defaultRuntimeOptions)
```

Comment:
Probably fine for now, but it's only a matter of time before someone creates an `EU` multi-regional bucket.

Reply:
We will validate that the region passed during workspace creation is a single region only and not multi-region like `EU` or `US`